paymentsheet-example/src/androidTest/java/com/stripe/android/lpm/TestAlma.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.lpm
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.stripe.android.BaseLpmTest
+import com.stripe.android.test.core.Currency
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class TestAlma : BaseLpmTest() {
+    private val alma = newUser.copy(
+        paymentMethod = lpmRepository.fromCode("alma")!!,
+        currency = Currency.EUR,
+        merchantCountryCode = "FR",
+    )
+
+    @Ignore("Complex authorization handling required")
```

Comment:
Can we remove this?

Reply:
This will redirect to Alma's page which we have no control of, so I can't really have a good test to catch this until they finish testing. Or I could remove this test.

Reply:
Do we have a test mode version of their page? We try to test these as thoroughly as possible.

Reply:
No, the current testmode page routes to their page, but the test below does test it successfully redirects.